examples/05-sage-list-compose-fleet/index.ts

```diff
@@ -0,0 +1,142 @@
+import { Program, BN } from '@project-serum/anchor';
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
+import bs58 from 'bs58';
+
+import { byteArrayToString, readAllFromRPC, readFromRPCOrError } from "@staratlas/data-source";
+import { Ship, Starbase, StarbasePlayer, SAGE_IDL } from "@staratlas/sage";
+
+import { newConnection, newAnchorProvider } from "../../shared/anchor-setup";
+import { loadKeypair } from "../../shared/wallet-setup";
+
+const SAGE_PROGRAM_ID = "SAGE2HAwep459SNq61LHvjxPk4pLPEJLoMETef7f7EE";
+
+const main = async (connection: Connection, myWallet: Keypair) => {
+    const args = process.argv.slice(2);
+    const x = args[0] || '0';
+    const y = args[1] || '0';
+
+    console.log('Example 05: Sage Ships available to compose this fleet');
+
+    const provider = newAnchorProvider(connection, myWallet);
+    const sageProgram = new Program(
+        SAGE_IDL,
+        SAGE_PROGRAM_ID,
+        provider,
+    );
+
+    const sageGameId = process.env.SAGE_GAME_ID || "11111111111111111111111111111111111111111111";
+    const sageGameIdPubkey = new PublicKey(sageGameId);
+
+    const sagePlayerProfileId = process.env.SAGE_PLAYER_PROFILE_ID || "11111111111111111111111111111111111111111111"
+    const playerPofilePubkey = new PublicKey(sagePlayerProfileId);
```

Comment:
Those defaults do not really make sense. I'd rather `throw` and make the environment non-optional. Alternatively, you could also fetch them via RPC. Game ID should be straight forward and profile could default to the first one if not specified otherwise. Let me know if you need examples for that.

Reply:
@mindrunner this just a convention that was been used in all examples to this point. I would prefer this PR get merged and then another be created to update all examples.